lib/execute.js

```diff
@@ -1,69 +1,129 @@
-const {
-  isAsync,
-  isCallbackFunction,
-  getPromisify,
-  log,
-  getErrorMessage
-} = require("./util");
+const { isAsync, isCallbackFunction, getPromisify, log, getErrorMessage } = require("./util");
 const metaData = Symbol("metaData");
 
 async function executeDescribe(describeBlock) {
-  console.log(`\n${describeBlock.title}`);
-
-  // Before
-  if (describeBlock.beforeHook) {
-    await executeHook("before", describeBlock.beforeHook);
-  }
-
-  // It blocks
-  for (const ic of describeBlock.itCollection) {
-    await executeItFlow(describeBlock, ic);
-  }
-
-  // After
-  if (describeBlock.afterHook) {
-    await executeHook("after", describeBlock.afterHook);
-  }
-}
+    console.log(describeBlock.title);
```

Comment:
why `console.log` here and `log` elsewhere?

Reply:
I'll move it to log. sorry about that.